src/app/loadout-drawer/loadout-drawer-reducer.ts

```diff
@@ -52,6 +52,11 @@ import {
  */
 export type LoadoutUpdateFunction = (loadout: Loadout) => Loadout;
 
+/** A helper for composing loadout update functions in the reducer */
```

Comment:
Would be good to say what "composing loadout update functions" means, like why you would use this. I think it just runs each provided update function in sequence?